control_server.go

```diff
@@ -368,7 +369,15 @@ func (this *ControlServer) HandleScript(w http.ResponseWriter, r *http.Request)
 	returnSuccess(w, r)
 }
 
-func (this *ControlServer) HandleConfig(w http.ResponseWriter, r *http.Request) {
+func (this *ControlServer) HandleConfigGet(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	err := json.NewEncoder(w).Encode(this.F.UpdatableConfig)
```

Comment:
Just the updatable ones? Or should we return all of it?

Reply:
just the updatable, this endpoint is added to improve the usability of the `POST /api/config`